domain/access/doc.go

```diff
@@ -4,4 +4,17 @@
 // Package access provides the services for managing users and permissions in
 // Juju. The users side is mainly concerned with authentication and the
 // permissions side with authorization.
+//
+// External users (i.e. users with an external domain in their user name e.g.
+// someone@external) can inherit permissions from the everyone@external user.
+//
```

Comment:
nitpick: Commas after "i.e.," and around "e.g.":

// External users (i.e., users with an external domain in their user name, e.g.,
// someone@external) can inherit permissions from the everyone@external user.